src/core/evaluator.js

```diff
@@ -387,6 +388,48 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
+    const standardFontNameToFileName = {
+      Courier: "FoxitFixed",
+      "Courier-Bold": "FoxitFixedBold",
+      "Courier-BoldOblique": "FoxitFixedBoldItalic",
+      "Courier-Oblique": "FoxitFixedItalic",
+      Helvetica: "FoxitSans",
+      "Helvetica-Bold": "FoxitSansBold",
+      "Helvetica-BoldOblique": "FoxitSansBoldItalic",
+      "Helvetica-Oblique": "FoxitSansItalic",
+      "Times-Roman": "FoxitSerif",
+      "Times-Bold": "FoxitSerifBold",
+      "Times-BoldItalic": "FoxitSerifBoldItalic",
+      "Times-Italic": "FoxitSerifItalic",
+      Symbol: "FoxitSymbol",
+      ZapfDingbats: "FoxitDingbats",
+    };
```

Comment:
It might not hurt to move this to https://github.com/mozilla/pdf.js/blob/master/src/core/standard_fonts.js instead!?